src/viking.js

```diff
@@ -1,11 +1,82 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor (health, strength) {
+        this.health = health;
+        this.strength = strength
+    }
+    attack () {
+        return this.strength
+    }
+    receiveDamage(damage) {
+        this.health = this.health - damage
+    }
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier { 
+    constructor(name, health, strength) {
+        super(health, strength) //Question for T.A.: does super always come before the arguments? When i tried switching this super line and the one after, it wasn't correct on Jasmine
```

Comment:
Yes, that's the order you should always use. 
`class`, `constructor`, and, if needed `super`